src/sentry/integrations/vsts/issues.py

```diff
@@ -38,28 +38,31 @@ def get_project_choices(self, group, **kwargs):
         project_choices = [(project["id"], project["name"]) for project in projects]
 
         params = kwargs.get("params", {})
-        defaults = self.get_project_defaults(group.project_id)
-        try:
-            default_project = params.get(
-                "project", defaults.get("project") or project_choices[0][0]
-            )
-        except IndexError:
-            return None, project_choices
-
-        # If a project has been selected outside of the default list of
-        # projects, stick it onto the front of the list so that it can be
-        # selected.
-        try:
-            next(True for r in project_choices if r[0] == default_project)
-        except StopIteration:
+        if group:
```

Comment:
@ceorourke can't we determine the default project to use based on the alert rule project? We can just pass that into the API, yea?

Reply:
@scefali I'm not sure what you mean, project here is the project in Azure.

Reply:
@ceorourke No, the Sentry project. We look up the defaults for the project attached to the group.

Reply:
@scefali ohh for what's passed into `get_project_defaults`? I thought you meant the return value.

Reply:
@ceorourke correct, pass in the project into `get_project_defaults`